rasa/shared/nlu/training_data/loading.py

```diff
@@ -60,7 +60,7 @@ def load_data(resource_name: Text, language: Optional[Text] = "en") -> "Training
     elif len(training_data_sets) == 1:
         training_data = training_data_sets[0]
     else:
-        training_data = training_data_sets[0].merge(*data_sets[1:])
+        training_data = training_data_sets[0].merge(*training_data_sets[1:])
```

Comment:
do you know why it was only failing on Windows?

Reply:
This was one of Anca's fixes, but I try to give my understanding:

For reference: a run where the  error that appeared: https://github.com/RasaHQ/rasa/runs/5556974140?check_suite_focus=true#step:15:843

```
    def test_dialogflow_data():
        td = load_data("data/examples/dialogflow/")
        assert not td.is_empty()
        assert len(td.entity_examples) == 5
        assert len(td.intent_examples) == 24
        assert len(td.training_examples) == 24
        assert len(td.regex_features) == 1
>       assert len(td.lookup_tables) == 2
E       AssertionError: assert 3 == 2
E        +  where 3 = len([{'elements': ['mexican', 'mexico', 'chinese', 'china', 'indian', 'india'], 'name': 'cuisine'}, {'elements': ['mexican...ina', 'indian', 'india'], 'name': 'cuisine'}, {'elements': ['centre', 'west', 'central', 'north'], 'name': 'location'}])
E        +    where [{'elements': ['mexican', 'mexico', 'chinese', 'china', 'indian', 'india'], 'name': 'cuisine'}, {'elements': ['mexican...ina', 'indian', 'india'], 'name': 'cuisine'}, {'elements': ['centre', 'west', 'central', 'north'], 'name': 'location'}] = <rasa.shared.nlu.training_data.training_data.TrainingData object at 0x000001FBC629AB48>.lookup_tables
```

So on Windows, the `load_data` function returned to many elements. 

The difference in the change seems to be: `training_data_sets: List[TrainingData] = [ds for ds in data_sets if ds]`

Therefore, this seems like it was a bug in this unit test, but not completely sure why it only surfaced on Windows

Reply:
🤔 yeah ok, still weird, but oh well